### PR TITLE
Throw if no workspace

### PR DESCRIPTION
**Pull request by yann300**
This throws an exception while using file manager and file provider if no workspace has been set.

### CODE REVIEW

File: apps/remix-ide/src/lib/gist-handler.js [hunk 1]
```diff
@@ -60,6 +60,8 @@ function GistHandler (_window) {
           if (!errorLoadingFile) {
             const provider = fileManager.getProvider('workspace')
             provider.lastLoadedGistId = gistId
+          } else {
+            modalDialogCustom.alert(errorLoadingFile.message || errorLoadingFile)
```
Comment (LianaHus) [inline apps/remix-ide/src/lib/gist-handler.js:64]: Please add a title to the modal, looks kind of unfinished without.

Comment (yann300) [reply apps/remix-ide/src/lib/gist-handler.js:64]: I can update those, but most of the custom alert doesn't set any title..

Comment (LianaHus) [reply apps/remix-ide/src/lib/gist-handler.js:64]: I know. I'm putting everywhere now. Looks better

File: apps/remix-ide/src/app/ui/modal-dialog-custom.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ var css = require('./styles/modal-dialog-custom-styles')
 module.exports = {
   alert: function (title, text) {
     if (text) return modal(title, yo`<div>${text}</div>`, null, { label: null })
-    return modal('', yo`<div>${title}</div>`, null, { label: null })
+    return modal('alert', yo`<div>${title}</div>`, null, { label: null })
```
Comment (LianaHus) [inline apps/remix-ide/src/app/ui/modal-dialog-custom.js:8]: capital?

**Review by Aniket-Engg — APPROVED** [review]

**Review by ioedeveloper — APPROVED** [review]

**Pull request merged**